tasktiger/tasktiger.py

```diff
@@ -591,6 +591,12 @@ def errored_tasks():
 
         return total_processed
 
+    def would_process_queue(self, queue_name):
+        """
+        Return if a queue_name would be processed by this tasktiger instance. It does not consider explicit
+        --queue param.
```

Comment:
@thomasst Ignoring the --queue param can be confusing and don't like it much. On the other hand, we would have to store the worker reference, and then use `worker.only_queues if worker else config['ONLY_QUEUES']` Thoughts?

Reply:
I would not store the worker reference on the `TaskTiger` object. 

It seems like the specific `--queue` parameters are currently specific to the worker. We could:

- Make the TaskTiger instance responsible for managing queues. For example, we could only initialize when we have to command line args (which might be tricky), or, have a function like  `set_queues_for_processing()` which is called to set/update the queues we want to process. The worker would then not store a set of queues but rely on the TaskTiger reference.
- Leave it as you did in the PR: The TaskTiger instance knows which queues it can process in general (from the config), but the Worker instance may have specific inclusions/exclusions.

The first option would be my preferred one but is also more work to do. @jkemp101 Do you have a preference here?

Also note max line length should be 79.

Reply:
Agreed with Joe to just rename to `would_process_configured_queue`

Ideal state would be workers reporting on which queues they are subscribed, and the monitoring agent would use that info instead of checking the config. But that would also be more work, so doing the simple thing now.